aten/src/ATen/native/native_functions.yaml

```diff
@@ -408,7 +408,7 @@
 - func: addr.out(Tensor self, Tensor vec1, Tensor vec2, *, Scalar beta=1, Scalar alpha=1, Tensor(a!) out) -> Tensor(a!)
   dispatch:
     CPU, CUDA: addr_out
-    Math: math_addr_out
+    CompositeImplicitAutograd: math_addr_out
```

Comment:
I think all those _out Math kernels should've been added to DefaultBackend instead? It's likely  due to we don't have a way to test when a kernel claims it supports multiple backends. 
But indeed for _out it's fine to stay in Math since we probably want to consider support autograd for them in the future? cc: @albanD

Reply:
Yeah, I think if they just call other at::_out functions, it's fine to keep them in Math. But really we need some strategy for testing that things are defined correctly